lib/rules/no-invalid-position-declaration/index.mjs

```diff
@@ -42,7 +59,10 @@ const rule = (primary) => {
 
 				// Nesting-supported at-rules only allow declarations when nested inside a rule
 				if (nestingSupportedAtKeywords.has(atRuleName)) {
-					const parentRule = findNodeUpToRoot(decl, ({ type }) => type === 'rule');
+					const parentRule = findNodeUpToRoot(
+						decl,
+						(node) => node.type === 'rule' || isIgnoredAtRule(node),
```

Comment:
[suggestion] Super-minor. We may want to use `isRule` that has already been imported into this file:

```suggestion
						(node) => isRule(node) || isIgnoredAtRule(node),
```